flake.nix

```diff
@@ -194,6 +194,7 @@
       microsoft-surface-laptop-amd = import ./microsoft/surface/surface-laptop-amd;
       microsoft-surface-common = import ./microsoft/surface/common;
       microsoft-surface-pro-3 = import ./microsoft/surface-pro/3;
+      microsoft-surface-pro-9 = import ./microsoft/surface-pro/9;
```

Comment:
Please also add this model to the table in the README. This is where the ci picks up model for evaluation.